src/mutation/transactWriteItems.ts

```diff
@@ -0,0 +1,197 @@
+import { DynamoDBDocumentClient, TransactWriteCommand, TransactWriteCommandOutput, TransactWriteCommandInput } from '@aws-sdk/lib-dynamodb';
+import { AnyObject, TableConfig, Key, ConditionExpressionInput, ConditionExpressionKind } from '../types';
+import { keyOperatorLookup } from '../query/queryBuilder';
+
+const buildConditionExpressions = (
```

Comment:
Looks very similar to buildConditionExpressions in https://github.com/oolio-group/dynamo-helper/blob/master/src/mutation/putItem.ts#L5